.env.test

```diff
@@ -58,7 +58,7 @@ POSTGRES_DB=redbox-core
 POSTGRES_PASSWORD=insecure
 CONTACT_EMAIL=redbox-copilot@cabinetoffice.gov.uk
 POSTGRES_HOST=localhost
-CORE_API_HOST=http://localhost
+CORE_API_HOST=testserver
```

Comment:
this is the trigger to mock the core-api

Reply:
This will break using `.env.test` to run the app locally, right?

Reply:
I've just checked - it does.